### PR TITLE
Improved clarity of open-source educational materials

### PR DESCRIPTION
**Pull request by moorepants**
- I swapped the order of the submission types to 1) match the order of
  the explanations below and 2) to make the content type more primary.
- I replaced "open-source learning modules" with "open-source educational
  materials" to make the documents consistent and only use one term.
- I expanded the definition of "open-source educational materials"
  with an attempt to be more clear and specific (because I'm still
  confused what the original wording meant).

### CODE REVIEW

**Comment by moorepants** [comment]
@labarba Can you have a look at this change? I may or may not be interpreting the original wording correctly, but hoping to improve clarity.

**Comment by moorepants** [comment]
@openjournals/jose-editors Would anyone be willing to give some feedback on this?

**Review by willingc — APPROVED** [review]
Looks good to me @moorepants. I will leave open for others to comment on and merge.

**Comment by moorepants** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ JOSE relies on the journal management infrastructure and tools developed for JOS
 
 JOSE publishes two types of (brief) articles that describe:
 
- 1. open educational software tools
+ 1. open-source educational materials
 
- 2. open-source learning modules
+ 2. open educational software tools
```
Comment (moorepants) [inline README.md]: Maybe this should be "open-source educational software tools" for more specificity.

**Comment by labarba** [comment]
> These documents should embody both course/lesson content and associated instructor notes.

I'm ambivalent about this language, which may be read as requesting separate instructor notes (even though the verb "embody" can be read to mean the content itself serves both purposes). I think what we do want to stress is that the materials should be _reusable_ for other instructors to readily adopt for their teaching.

**Comment by moorepants** [comment]
I've updated the text based on Lorena's feedback. Ready for review.

**Pull request merged**